src/content/extensions/multipleroutes/src/README.md

```diff
@@ -3,11 +3,11 @@ Configures the WebRTC traffic routing options in Chrome's privacy settings.
 
 ★ What it does:
 This configures WebRTC to not use certain IP addresses or protocols:
-- IP addresses not visible to the public internet (e.g. addresses like 192.168.1.2)
+- private IP addresses not visible to the public internet (e.g. addresses like 192.168.1.2)
 - any public IP addresses associated with network interfaces that are not used for web traffic (e.g. an ISP-provided address, when browsing through a VPN)
 - Require WebRTC traffic to go through proxy servers as configured in Chrome. Since most of the proxy servers don't handle UDP, this effectively turns off UDP until UDP proxy support is available in Chrome and such proxies are widely deployed.
  
-Once the extension is installed, WebRTC will only use public IP addresses associated with the interface used for web traffic, typically the same addresses that are already provided to sites in browser HTTP requests.
+Once the extension is installed on Chrome versions prior to M48, WebRTC will only use the public IP address associated with the interface used for web traffic, typically the same addresses that are already provided to sites in browser HTTP requests. For Chrome version M48 and after, this extension provides one more configuration which allows WebRTC to use both the default public address and, for machines behind a NAT, the default private address which is associated with the public one. Said behavior will be the default after a fresh installation of the extension to Chrome M48. For upgrade scenarios, the previous selected configuration should not be changed.
```

Comment:
Done.

Thanks,
Guowei

On Thu, Jan 7, 2016 at 12:28 AM, Christoffer Jansson <
notifications@github.com> wrote:

> In src/content/extensions/multipleroutes/src/README.md
> <https://github.com/webrtc/samples/pull/727#discussion_r49050426>:
>
> >  - any public IP addresses associated with network interfaces that are not used for web traffic (e.g. an ISP-provided address, when browsing through a VPN)
> >  - Require WebRTC traffic to go through proxy servers as configured in Chrome. Since most of the proxy servers don't handle UDP, this effectively turns off UDP until UDP proxy support is available in Chrome and such proxies are widely deployed.
> >
> > -Once the extension is installed, WebRTC will only use public IP addresses associated with the interface used for web traffic, typically the same addresses that are already provided to sites in browser HTTP requests.
> > +Once the extension is installed on Chrome versions prior to M48, WebRTC will only use the public IP address associated with the interface used for web traffic, typically the same addresses that are already provided to sites in browser HTTP requests. For Chrome version M48 and after, this extension provides one more configuration which allows WebRTC to use both the default public address and, for machines behind a NAT, the default private address which is associated with the public one. Said behavior will be the default after a fresh installation of the extension to Chrome M48. For upgrade scenarios, the previous selected configuration should not be changed.
>
> Consider replacing Once with either If or When.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/webrtc/samples/pull/727/files#r49050426>.
>